src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -592,6 +592,29 @@ static public Size GetScreenSize()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the geometry hit-testing enabled or disabled for the application.
+        /// </summary>
+        /// <param name="enable">True to enable geometry hit-testing, false otherwise.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public void SetGeometryHittestEnabled(bool enable)
+        {
+            Interop.Application.SetGeometryHittestEnabled(enable);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
```suggestion
            NDalicPINVOKE.ThrowExceptionIfExist();
```

Please use more shortten method.